lib/models/environment.py

```diff
@@ -14,3 +14,7 @@ class Env(BaseModel, frozen=True):
     date: Optional[datetime.datetime] = (
         datetime.datetime.today() + datetime.timedelta(days=1)
     )
+
+    @property
+    def env_id(self) -> str:
+        return str(hash(self))
```

Comment:
@coderabbitai how can I use UUIDs in fastAPI? is it similar to what we do in DJANGO?

Reply:
Currently, we don't use UUID because there is an embedded logic that assumes many use cases may be similar/duplicated, so we hash the entire object in an attempt to spare storage.

We just need to think on a better hashing algorithm (not rely on python native), because it needs to be session agnostic and thread-safe.